torch/_dynamo/eval_frame.py

```diff
@@ -79,7 +79,7 @@ class Unset(Enum):
 DONT_WRAP_FILES = {
     # For tracing into fx modules
     inspect.getsourcefile(GraphModule),
-    join(dirname(dirname(__file__)), "onnx/_internal/fx/dynamo_exporter.py"),
```

Comment:
yikes this needs additional approval now lol. I wonder if it makes sense to include whole 'onnx/_internal/fx` directory or not 🤔. Couldn't tell since really have little idea what this is doing.

Reply:
@kit1980 or @malfet maybe could also help.